src/backend/index.js

```diff
@@ -4,10 +4,14 @@ import { initBroadCastEvents } from "./broadcastQueries";
 import { initLinkEvents } from "./links";
 import { checkVersions } from "./checkVersions";
 
-// hook should have been injected before this executes.
-const hook = window.__APOLLO_DEVTOOLS_GLOBAL_HOOK__;
+let hook;
 let bridge;
 let connected;
+let storage;
+
+export const sendBridgeReady = () => {
```

Comment:
Would you mind adding a comment here to explain why this function exists and is exported instead of inlined into the call site 🙏? I assume this is so the react native debugger can import and call this :)